### PR TITLE
add Heveloon ltd

### PR DESCRIPTION
**Pull request by samuel20miglia**

### CODE REVIEW

**Review by edoardocostantinidev — APPROVED** [review]

**Comment by Sonic0** [comment]
I think there is a typo in tags list: `Noda.js`

File: data/heveloon.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "remote_policy": "Full",
+  "hiring_policy": "Contract",
+  "tags": [
+    "React",
+    "Flutter",
+    "AWS",
+    "IoT",
+    "Swifth",
+    "Kotlin",
+    "Noda.js",
```
Comment (edoardocostantinidev) [inline data/heveloon.json:18]:
```suggestion
    "Node.js",
```

**Review by edoardocostantinidev — CHANGES_REQUESTED** [review]

File: data/heveloon.json (hunk 1 above)
Comment (samuel20miglia) [reply data/heveloon.json]: done

**Review by samuel20miglia — COMMENTED** [review]
done

**Review by edoardocostantinidev — APPROVED** [review]

**Pull request merged**